docusaurus/docs-internals/editor-controls/multivalue-controls/gallery.md

```diff
@@ -162,43 +251,127 @@ In this example, the user cannot remove the last item in the gallery.
 }
 ```
 
-### Return value
+#### Default value examples
 
-Returns a stringified array of objects with the following values:
+In this example, setting a default value for the gallery control.
 
 ```js
 {
+  id: "slides",
+  type: "gallery",
+  default: {
+    value: '[
+      {
+        id: "1",
+        uid:"dbe6479df22c4ef2ac6553aa241d2c5c.jpg",
+        fileName: "image1.jpg",
+        width: 800,
+        height: 600
+      },
+      {
+        id: "2",
+        uid:"fbe6479df22c4ef2ac6555te241d2c5c.jpg",
+        fileName: "image2.jpg",
+        width: 1024,
+        height: 768
+      }
+    ]'
+  }
+}
+```
+
+#### CSS examples
+
+In this example shows how to apply basic styles to the gallery images.
+
+
+```js
+{
+  id: "slides",
+  type: "gallery",
+  style: ({ value }) => {
+    const { width, height } = value;
+    
+    return {
+      "{{WRAPPER}} .brz-gallery img": {
+        width: `${width}px`,
+        height: `${height}px`
+      },
+    };
+  },
+};
+```
+
+
+#### Usage in HTML example
+
+```tsx
+import { Brizy } from "@brizy/core";
+import { Gallery } from "./Gallery";
+import React, { JSX } from "react";
+
+interface ImageProps {
   id: number;
-  uid: string;
   fileName: string;
   width: number;
   height: number;
 }
-```
-Example:
-```js
-{
-  value: '[
-    {
-      "id": 1,
-      "uid": "dbe3a79df22c4ef2ac6553aa241d2c5c.jpg",
-      "fileName": "image1.jpg",
-      "width": 1600,
-      "height": 1042
-    },
-    {
-      "id": 2,
-      "uid": "a7c45ea83554c81024b84d724fd8286d.jpg",
-      "fileName": "image2.jpg",
-      "width": 640,
-      "height": 359
-    }
-  ]'
+
+interface Props {
+  galleryImages: Array<ImageProps>;
 }
-```
 
-`id` - The index of the image in the gallery; <br/>
-`uid` - The unique identifier of the image; <br/>
-`fileName` - The unique identifier of the image; <br/>
-`width` - The width of the image; <br/>
-`height` - The height of the image; <br/>
+const Gallery = (props: Props): JSX.Element => {
+  const { galleryImages } = props;
+
+  const attributes = {
+    "data-gallery": galleryImages.length > 0
+  };
+  
+  return (
+    <div className="brz-gallery" {...attributes}>
+      {galleryImages.length > 0 && (
+        galleryImages.map((image) => (
+          <Image 
+            key={image.id} 
+            fileName={image.fileName} 
+            width={image.width} 
+            height={image.height} 
+          />
+        ))
+      )}
+    </div>
+  );
+};
```

Comment:
rewrite example, because your `gallery` return an string, which is strigified array of images